aptos-move/aptos-aggregator/src/aggregator_change_set.rs

```diff
@@ -0,0 +1,507 @@
+// Copyright © Aptos Foundation
+
+use crate::{
+    aggregator_extension::{
+        validate_history, AggregatorID, AggregatorState, DeltaHistory, SpeculativeStartValue,
+    },
+    bounded_math::{abort_error, addition_deltavalue, ok_overflow, ok_underflow},
+};
+use aptos_state_view::StateView;
+use move_binary_format::errors::{PartialVMError, PartialVMResult};
+use move_core_types::vm_status::{StatusCode, VMStatus};
+
+/// When merging aggregator changes of two transactions,
+/// unable to merge the histories of the transactions.
+pub(crate) const EMERGE_HISTORIES: u64 = 0x02_0008;
```

Comment:
We can keep this for now, but this error should never be propagated to the user. I think I mentioned during one of the syncs about aggregator errors, and this is somewhat what I meant (with what @igor-aptos also partially added in bounded math)

```
enum AggregatorError {
  UnmergableHistories,
  ....
}
```
so we can use this over VMStatus. But let's revisit this later👍